src/main/java/uk/gov/register/views/HomePageView.java

```diff
@@ -51,7 +51,7 @@ public HomePageView(
 
     @SuppressWarnings("unused, used from template")
     public String getRegisterText() {
-        return markdownProcessor.markdown(getRegister().getText());
```

Comment:
I think we should leave this here? We still want to render `text` fields as markdown if we can.

Reply:
We only expect a string in the frontend so that we can control the styling better. Why do we want markdown?

Reply:
It's only for fields that are of datatype `text` because we specify in the datatype register that that datatype can contain markdown https://datatype.register.gov.uk/record/text. However I think this might be broken in prod currently anyway 😕